### PR TITLE
Step assemblies: Import

### PR DESCRIPTION
**Pull request by WirajanDASILVA**
* **Please check if the PR fulfills these requirements**
- [X] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix:
- [X] New features:
- [ ] Documentation
- [ ] Optimization

* **Other information**:
to be merged after the the holes in #284

### CODE REVIEW

File: volmdlr/step.py [hunk 1]
```diff
@@ -183,23 +188,40 @@ def advanced_brep_shape_representation(arguments, object_dict):
...
+            for frame3d in shape_representation_frames:  # shape_representation
+                if f3d.origin != volmdlr.O3D != frame3d.origin and frame3d != f3d:
+                    break
+            else:
+                continue
+            break
+        else:
+            continue
+        new_faces = [face.frame_mapping(frame3d, 'new') for face in shell3d.faces]
+        shell3d.faces = new_faces
```
Comment (AxelRinghausen) [inline volmdlr/step.py:203]: I would recommend not using for/else as it is not very readable.

File: volmdlr/wires.py [hunk 2]
```diff
@@ -1671,6 +1755,11 @@ def cut_by_line(self, line: volmdlr.edges.Line2D) -> List['Contour2D']:
         if not intersections or len(intersections) < 2:
             return [self]
         if len(intersections) % 2 != 0:
+            ax = self.plot()
+            line.plot(ax=ax)
+            [i[0].plot(ax=ax) for i in intersections]
+            self.save_to_file('/home/axel/Bureau/contour2d')
+            line.save_to_file('/home/axel/Bureau/line2d')
```
Comment (AxelRinghausen) [inline volmdlr/wires.py:1762]: Hmm... sorry about that. 
```suggestion
```

**Review by AxelRinghausen — APPROVED** [review]

**Review by AxelRinghausen — DISMISSED** [review]

File: volmdlr/step.py (hunk 1 above)
Comment (WirajanDASILVA) [reply volmdlr/step.py:203]: I understand. Then a better solution would be using a variable?

**Review by masfaraud — DISMISSED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### New Features
 * Unit coversion factor parameter added to the end of the from_step arguments parameter.
 So we can convert the units correctly
+* SphericalSurfcae3D: rotation, translation, frame_mapping
```
Comment (GabrielJMS) [inline CHANGELOG.md:13]:
```suggestion
* SphericalSurface3D: rotation, translation, frame_mapping
```

**Review by GabrielJMS — APPROVED** [review]

**Pull request merged**